pgcli/packages/pgspecial.py

```diff
@@ -762,7 +762,14 @@ def sql_name_pattern(pattern):
 def show_help(cur, arg, verbose):  # All the parameters are ignored.
     headers = ['Command', 'Description']
     result = []
-    for command, value in sorted(CASE_SENSITIVE_COMMANDS.iteritems()):
+
+    try:
+        itr = CASE_SENSITIVE_COMMANDS.iteritems()
+    except AttributeError:
+        #Python 3
+        itr = CASE_SENSITIVE_COMMANDS.items()
+
+    for command, value in sorted(itr):
```

Comment:
You can just replace this line with:

```
for command, value in sorted(CASE_SENSITIVE_COMMANDS.items()):
```

In Python 2 it will return a list and Python 3 it'll be an iterator, but both of them are acceptable for `sorted()`.